tools/functional-tester/etcd-tester/tester.go

```diff
@@ -235,3 +257,20 @@ func isSameValueInMap(hashes map[string]int64) bool {
 	}
 	return ok
 }
+
+func (c *cluster) compactKV(rev int64) error {
```

Comment:
we do not need to send compact request to every member. this request will be replicated by raft anyway...